lackey/PlatformManagerDarwin.py

```diff
@@ -328,7 +328,7 @@ def getWindowRect(self, hwnd):
                 width = w["kCGWindowBounds"]["Width"]
                 height = w["kCGWindowBounds"]["Height"]
                 return (x, y, width, height)
-        raise OSError("Unrecognized window number {}".format(hwnd))
+        return None
```

Comment:
This particular method should raise a descriptive error. Returning None when the recipient is expecting a tuple will just raise a different (less clear) error.